### ISSUE TITLE
Defined without required GraphQL definition for 'Float' in __requires__ exception

### ISSUE THREAD

**Issue by kudamhazo** [mirumee/ariadne-graphql-modules#1]
I am not sure if this is intended behavior but isn't `Float` a built-in type of GraphQL that shouldn't require to be passed in as `__requires__` attribute of object type class. Here is my object type definition that is failing:

```python
class UomType(ObjectType):
    __schema__ = gql(
        """
        type UnitOfMeasure {
            id: ID!
            code: Int!
            name: String!
            uomType: String
            factorB: Float
            factorC: Float
            remarks: String
            abbrev: String
        }
        """
    )
```

This fails with error:

`ValueError: UomType class was defined without required GraphQL definition for 'Float' in __requires__`

**Comment by rafalp** [comment]
Yup, it is! `Float` needs to be added to `GRAPHQL_TYPES = (...)` tuple in `dependencies` module. Would you like to open PR with fix?

**Comment by rafalp** [comment]
Released in 0.3.0 🎉
